application/server/execute_command.go

```diff
@@ -37,22 +39,24 @@ func executeCommandHandler(srv *jrpc2.Server) jrpc2.Handler {
 		// so we don't want to propagate it to functions that start background operations
 		bgCtx := context.Background()
 		method := "ExecuteCommandHandler"
+		c := config.CurrentConfig()
+		learnService := learn.New(c, c.Engine().GetNetworkAccess().GetUnauthorizedHttpClient)
```

Comment:
question: is it correct to use GetUnauthorizedHttpClient() why shouldn't we use a client with authorization?

Reply:
Learn doesn't need authorization :)